app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -354,6 +357,46 @@ class StateFragmentPresenter @Inject constructor(
     itemList.add(contentViewModel)
     stateAdapter.notifyDataSetChanged()
   }
+  private fun checkAndAddInteraction() {
+    if (currentEphemeralState.stateTypeCase.number == EphemeralState.PENDING_STATE_FIELD_NUMBER) {
+      when (currentEphemeralState.state.interaction.id) {
+        MULTIPLE_CHOICE_INPUT -> {
+          addMultipleChoiceInputItem()
```

Comment:
Since this is for both, maybe use 'addSelectionInteraction()' or something similar to indicate both interactions are being represented together.

Reply:
done.